tests/bad_svd.rs

```diff
@@ -4,7 +4,7 @@ use svd_parser as svd;
 fn arm_sample_faulty() {
     let xml = include_str!(concat!(
         env!("CARGO_MANIFEST_DIR"),
-        "/tests/ARM_Sample_faulty.svd"
+        "/tests/data/ARM_Sample_faulty.svd"
```

Comment:
you missed this I think

Reply:
You mean reverting it as it was? I think that this structure is more intuitive (keep tests and data separate)... it's a minimal change and tests keep passing as they were anyway?:

```rust
$ cargo test
(...)
running 1 test
test arm_sample_faulty ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
```

Reply:
Sorry, I haven't seen you moved the file!